### PR TITLE
docs: add Exherbo Linux install instructions in README.md

### PR DESCRIPTION
**Pull request by davlgd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,6 +178,14 @@ For Arch Linux you can download `gitoxide` from `community` repository:
 pacman -S gitoxide
 ```
 
+### Download from Exherbo Linux rust repository
+
+For Arch Linux you can download `gitoxide` from [rust](https://gitlab.exherbo.org/exherbo/rust/-/tree/master/packages/dev-scm/gitoxide) repository:
```
Comment (Byron) [inline README.md:183]: Could you review this sentence and make sure it's what is intended?

File: README.md [hunk 2]
```diff
@@ -178,6 +178,14 @@ For Arch Linux you can download `gitoxide` from `community` repository:
...
+```
+cave resolve -x repository/rust
+cave resove -x gitoxide
```
Comment (Byron) [inline README.md:186]: This is probably a typo.

**Review by Byron — CHANGES_REQUESTED** [review]
Thanks for distributing `gitoxide`!

I hope there is a little time for the fixes, so the PR can be merged.

File: README.md (hunk 2 above)
Comment (davlgd) [reply README.md]: Sorry for that, some checks were missed 😅

**Pull request merged**